packages/tracing/src/transaction.ts

```diff
@@ -68,11 +69,14 @@ export class Transaction extends SpanClass implements TransactionInterface {
   }
 
   /**
-   * Set observed measurements for this transaction.
+   * Set observed measurement for this transaction.
+   * @param name Name of the measurement
+   * @param value Value of the measurement
+   * @param unit Unit of the measurement. (Defaults to 'ms' - milliseconds)
    * @hidden
    */
-  public setMeasurements(measurements: Measurements): void {
-    this._measurements = { ...measurements };
+  public setMeasurement(name: string, value: number, unit: MeasurementUnit = 'ms'): void {
```

Comment:
The default should be an empty string, `ms` is pretty unexpected behaviour.

Reply:
Now that I know units are basically free form text, I agree. Changed in dd9f8c80a4a222a0a5750483821977ab52fcf099.